services/ResourceService/templates/getResourceByUUIDfromTS.sparql

```diff
@@ -0,0 +1,5 @@
+PREFIX nfo: <http://www.semanticdesktop.org/ontologies/2007/03/22/nfo/v1.2/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+SELECT ?s WHERE {
+  ?s nfo:uuid "{{uuid}}"^^<http://www.w3.org/2001/XMLSchema#string> .
```

Comment:
i could stick here an extra variable, to get also type of resource. This way we can avoid hitting fedora if the send one mismatches the existing one. (it's rdf, we try to put a binary for example).